event_test.go

```
@@ -4,20 +4,20 @@ import (
 
 type testListener struct {
 	userData string
 }
 
-func (l *testListener) Handle(e event.Event) error {
-	if ret := e.Get("result"); ret != nil {
+func (l *testListener) Handle(e event.Event[event.M]) error {
+	if ret := e.Data()["result"]; ret != nil {
 		str := ret.(string) + fmt.Sprintf(" -> %s(%s)", e.Name(), l.userData)
-		e.Set("result", str)
+		e.SetData(event.M{"result": str})
```

golangci-lint (1.64.8)

Error return value of `e.SetData` is not checked

(errcheck)